test.js

```diff
@@ -1,105 +1,105 @@
-'use strict';
+// merge-stream
```

Comment:
Why do we need this comment here?

Reply:
It's not needed... it's just a very minimal header at the top of the file to give the developer a tiny bit of context when they are working on multiple projects at the same time.  If you don't like it, it totally can be removed no problem.